README.md

```diff
@@ -1,18 +1,20 @@
 # DC/OS Tools, Packages, and Installers for various platforms
 
-Tools for building DC/OS and launchign a cluster with it in the hardware of a customer's choice.
+Tools for building DC/OS and launching a cluster with it on a hardware of user's choice.
 
-  - *docker/* Locally defined docker containers packages are built in
+  - *docker/* Locally defined docker containers packages are built
```

Comment:
we don't have a hard wrap limit in the markdown / this shouldn't wrap.

Also note the dockers aren't actually built in this folder. The folder simply contains Dockerfile definitions. Each dockerfile is in a folder which has the name that the particular builder should use (ex: dcos-builder). When people say "docker": "dcos-builder" these dockers will get lazily built when requested. They'll then be used to build the package which asked for that particular docker as a build environment.